fastlane/lanes/release.rb

```diff
@@ -312,9 +312,13 @@ def check_pods_references
   # This will also print the result to STDOUT
   result = ios_check_beta_deps(lockfile: File.join(PROJECT_ROOT_FOLDER, 'Podfile.lock'))
 
-  style = result[:pods].nil? || result[:pods].empty? ? 'success' : 'warning'
-  message = "### Checking Internal Dependencies are all on a **stable** version\n\n#{result[:message]}"
-  buildkite_annotate(context: 'pods-check', style: style, message: message) if is_ci
+  return unless is_ci
+
+  all_dependencies_stable = result[:pods].nil? || result[:pods].empty?
```

Comment:
I did this tidy because I thought I'd add an emoji an wanted to DRY the check.

I decided not to add the emoji to the text because Buildkite already has icons and colors for each annotation style. But I kept the code like this because I think it's a tiny readability improvements.

Also, since I'm on my way to write more commentary than code, I considered being explicit in the variable names and the message in mentioning CocoaPods. But, I'd say leaving it as generic "dependencies" makes it easier for us to update the logic to incorporate something that parses the `Package.resolved` JSON looking for dependencies that point to non stable versions.